### PR TITLE
Feature/feature convert utc

### PR DESCRIPTION
**Pull request by AlvarTB**
ConvertUTC file added

### CODE REVIEW

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+public class UTCLocalConverter {
+    public static String convertUTCtoLocal(String datein) throws ParseException {
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
+        Date date = sdf.parse(datein);
+        sdf.setTimeZone(TimeZone.getDefault());
+        return sdf.format(date);
+    }
+
+    public static String convertLocaltoUTC(String datein) throws ParseException{
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:17]: Codacy Issue found: ['{' is not preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           

Complexity increasing per file
==============================
- src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555&bid=18116369)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:18]: Codacy Issue found: [When instantiating a SimpleDateFormat object, specify a Locale](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        TimeZone tz = TimeZone.getDefault();
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:19]: Codacy Issue found: [Avoid unused local variables such as 'tz'.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
+package org.pesmypetcare.webservice.entity;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class UTCLocalConverter {
+    public static String convertUTCtoLocal(String datein) throws ParseException {
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:10]: Codacy Issue found: [The String "yyyy-MM-dd'T'HH:mm:ss" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    public static String convertLocaltoUTC(String datein) throws ParseException{
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+        TimeZone tz = TimeZone.getDefault();
+        sdf.setTimeZone(TimeZone.getDefault());
+        Date date = sdf.parse(datein);
+        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
+        return sdf.format(date);
+    }
+
+    public static String getCurrentUTC(){
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:26]: Codacy Issue found: ['{' is not preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class UTCLocalConverter {
+    public static String convertUTCtoLocal(String datein) throws ParseException {
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:11]: Codacy Issue found: [The String "UTC" appears 3 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 7]
```diff
@@ -0,0 +1,32 @@
+package org.pesmypetcare.webservice.entity;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class UTCLocalConverter {
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:8]: Codacy Issue found: [Utility classes should not have a public or default constructor.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+        TimeZone tz = TimeZone.getDefault();
+        sdf.setTimeZone(TimeZone.getDefault());
+        Date date = sdf.parse(datein);
+        sdf.setTimeZone(TimeZone.getTimeZone("UTC"));
+        return sdf.format(date);
+    }
+
+    public static String getCurrentUTC(){
+        SimpleDateFormat sdf = new SimpleDateFormat("dd/MM/yyyy'T'HH:mm:ss");
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:27]: Codacy Issue found: [When instantiating a SimpleDateFormat object, specify a Locale](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555&bid=18116369)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 9]
```diff
@@ -0,0 +1,35 @@
+package org.pesmypetcare.webservice.entity;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class UTCLocalConverter {
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:8]: Codacy Issue found: [Utility classes should not have a public or default constructor.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+    public static String convertLocaltoUTC(String datein) throws ParseException {
+        SimpleDateFormat sdf = new SimpleDateFormat(DATEFORMAT);
+        sdf.setTimeZone(TimeZone.getDefault());
+        Date date = sdf.parse(datein);
+        sdf.setTimeZone(TimeZone.getTimeZone(UTC));
+        return sdf.format(date);
+    }
+
+    public static String getCurrentUTC() {
+        SimpleDateFormat sdf = new SimpleDateFormat(DATEFORMAT);
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:30]: Codacy Issue found: [When instantiating a SimpleDateFormat object, specify a Locale](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555&bid=18116369)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 11]
```diff
@@ -0,0 +1,36 @@
+package org.pesmypetcare.webservice.entity;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+class UTCLocalConverter {
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:9]: Codacy Issue found: [Utility classes should not have a public or default constructor.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 12]
```diff
@@ -0,0 +1,36 @@
+package org.pesmypetcare.webservice.entity;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:6]: Codacy Issue found: [Unused import - java.util.Locale.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+    public static String convertLocaltoUTC(String datein) throws ParseException {
+        SimpleDateFormat sdf = new SimpleDateFormat(DATEFORMAT);
+        sdf.setTimeZone(TimeZone.getDefault());
+        Date date = sdf.parse(datein);
+        sdf.setTimeZone(TimeZone.getTimeZone(UTC));
+        return sdf.format(date);
+    }
+
+    public static String getCurrentUTC() {
+        SimpleDateFormat sdf = new SimpleDateFormat(DATEFORMAT);
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:31]: Codacy Issue found: [When instantiating a SimpleDateFormat object, specify a Locale](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

**Review by oricat8 — DISMISSED** [review]

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555&bid=18116369)

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+    public static String convertUTCtoLocal(String datein) throws ParseException {
+        SimpleDateFormat sdf = new SimpleDateFormat(DATEFORMAT);
+        sdf.setTimeZone(TimeZone.getTimeZone(UTC));
+        Date date = sdf.parse(datein);
+        sdf.setTimeZone(TimeZone.getDefault());
+        return sdf.format(date);
+    }
+
+    public static String convertLocaltoUTC(String datein) throws ParseException {
+        SimpleDateFormat sdf = new SimpleDateFormat(DATEFORMAT);
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java:27]: Codacy Issue found: [When instantiating a SimpleDateFormat object, specify a Locale](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555&bid=18116369)

**Review by oricat8 — DISMISSED** [review]

File: src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+        sdf.setTimeZone(TimeZone.getTimeZone(UTC));
+        return sdf.format(date);
+    }
+
+    public static String getCurrentUTC() {
+        SimpleDateFormat sdf = new SimpleDateFormat(DATEFORMAT, Locale.getDefault());
+        Date date2 = new Date();
+        sdf.setTimeZone(TimeZone.getTimeZone(UTC));
+        return sdf.format(date2);
+    }
```
Comment (santidrj) [inline src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java]: Missing javadoc

**Review by santidrj — CHANGES_REQUESTED** [review]
There are some changes to do

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/main/java/org/pesmypetcare/webservice/entity/UTCLocalConverter.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5527555&bid=18116369)

**Review by oricat8 — APPROVED** [review]

**Review by santidrj — APPROVED** [review]
:+1:

**Pull request merged**